### ISSUE TITLE
[5.0] Session related functionality fails in codeception functional tests

### ISSUE THREAD

**Issue by janhenkgerritsen** [laravel/framework#6138]
See the discussion at https://laracasts.com/discuss/channels/general-discussion/latest-v5-laravelframework-csrfmiddleware-changes-broke-codeception-functional-tests.

**Comment by GrahamCampbell** [comment]
Fixed in 5.0.2. :)

**Comment by vladprokopchuk** [comment]
doesnot  work after 

// App/Providers/TestingServiceProvider.php

<?php namespace App\Providers;

use Illuminate\Support\ServiceProvider;

class TestingServiceProvider extends ServiceProvider {

     /**
      * Register the service provider.
      *
      * @return void
      */
    public function register()
    {
        if ($this->app->environment() == 'testing')
        {
            $this->app['config']['session.driver'] = 'native';
        }
    }

}

the same problem test fuctional test with CSRF middleware doesnot work

**Comment by jaumeavila** [comment]
Hi, 
I have done the TestingServiceProvider as explained, and it is registered correctly but it keeps failing.
If I set a session variable in the test CEST file, inside the test method I can retrieve it, but once I try to get it in the Laravel controller, it is just not there.
And if I login the Sentry user in the test just like
`Sentry::login(Sentry::findUserById($u->id));`

It works in the Authcontroller and I can get it using Sentry::getUser(), but once a redirect is done, it is no longer there.

Any possible solution?

Thanks,
Jaume

**Comment by janhenkgerritsen** [comment]
Please do not reply to closed issues, if you still have a problem you should open a new issue and explain your problem.

Furthermore, this issue belongs to the Laravel Framework repository and I think you want to report a problem with Codeception, so you should create an issue at the Codeception repository.

If you decide to create an issue on the Codeception repository, please provide additional details like the version of Codeception you are using, your Codeception configuration and anything else you can think of that can be useful.
